ambry-router/src/main/java/com/github/ambry/router/NonBlockingRouterMetrics.java

```diff
@@ -569,8 +571,12 @@ public NonBlockingRouterMetrics(ClusterMap clusterMap, RouterConfig routerConfig
 
     requestsWithUnknownQuotaResourceRate = metricRegistry.meter(
         MetricRegistry.name(QuotaAwareOperationController.class, "RequestsWithUnknownQuotaResourceRate"));
-    nonQuotaCompliantRequestRate =
+    rejectedRequestRate =
```

Comment:
Is it okay for the name of the metric to remain 'NonQuotaCompliantRequestRate'?

Reply:
fixed.